<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Andizhizucheng2</class>
 <widget class="QWidget" name="Andizhizucheng2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,5,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimages.qrc">
         <normaloff>:/image/images/4g.png</normaloff>:/image/images/4g.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimages.qrc">
         <normaloff>:/image/images/wireless.svg</normaloff>:/image/images/wireless.svg</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimages.qrc">
         <normaloff>:/image/images/bl.svg</normaloff>:/image/images/bl.svg</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimages.qrc">
         <normaloff>:/image/images/locate.svg</normaloff>:/image/images/locate.svg</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit_3"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(153, 211, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>按地址组秤</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>传感器个数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sensor_number"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>01</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_1">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>02</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_2">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>03</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_3">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="3">
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>04</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_4">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>05</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_5">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>06</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_6">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>07</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_7">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="3">
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>08</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_8">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_25">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>09</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_9">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_23">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>10</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_10">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_21">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>11</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_11">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="3">
            <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>12</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_12">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_33">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>13</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_13">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_31">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>14</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_14">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_29">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>15</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_15">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="3">
            <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_27">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(167, 167, 167);</string>
               </property>
               <property name="text">
                <string>16</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sensor_16">
               <property name="text">
                <string>无</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="returnIndex">
       <property name="font">
        <font>
         <family>AR PL UMing TW MBE</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/images/index.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimages.qrc">
         <normaloff>:/image/images/index.svg</normaloff>:/image/images/index.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="returnPage">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimages.qrc">
         <normaloff>:/image/images/shangyiye.svg</normaloff>:/image/images/shangyiye.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myimages.qrc"/>
 </resources>
 <connections/>
</ui>
